### ISSUE TITLE
Select all messages in folder

### ISSUE THREAD

**Issue by fnetX** [nextcloud/mail#4285]
### Feature Request

Provide an option to select all mail in the current folder.

#### Summary

Currently, it is possible to select multiple messages one-by-one. Most e-mail clients have an option to select all e-mail e.g. for (re-)moving them. Please add this option to Nextcloud / Mail as well.

**Comment by StCyr** [comment]
Yes. What about creating a "Select All" action in the multi-select menu? So you'd have to select 1 message first, then go to the multi-select menu and there select "Select All". ping @jancborchardt 

That being said, in the mean time, you can already select multiple messages at once by pressing the SHIFT key when selecting a message; No need to select all messages one-by-one.

**Comment by dermitdemdino** [comment]
@StCyr : A great tip, thank you very much.

**Comment by jakobroehrl** [comment]
Great tip, maybe we should add it to the keyboard shortcut list?
